### ISSUE TITLE
Config: Automatically update the CSP when a CDN is configured

### ISSUE THREAD

**Issue by lastzero** [photoprism/photoprism#3367]
**As a user who has a CDN configured, I would like the Content Security Policy to be updated automatically so that content from the CDN is not blocked by the browser.**

- https://github.com/photoprism/photoprism/discussions/3358

We should also make the CSP config option available in our Community Edition so that the reverse proxy settings do not need to be touched.

**Comment by lastzero** [comment]
Duplicate of #3454.
